bindings/rust/src/bindings/mod.rs

```diff
@@ -406,6 +527,31 @@ impl KZGProof {
             }
         }
     }
+
+    pub fn verify_sample_kzg_proof(
+        sample: Sample,
+        index: usize,
+        commitment_bytes: Bytes48,
+        proof_bytes: Bytes48,
```

Comment:
My fork of c-kzg-4844 is a bit outdated. In upstream these are passed as references. I think it would be good to mirror that here. So change `Sample` to `&Sample`, etc.